src/ResourceManagement/ResourceManager/Microsoft.Azure.Management.ResourceManager.Fluent.csproj

```diff
@@ -5,18 +5,17 @@
 
   <PropertyGroup>
     <Description>Provides resource group and resource management (Fluent) capabilities for Microsoft Azure.</Description>
-    <Version>1.13.0</Version>
+    <Version>1.14.0</Version>
     <AssemblyName>Microsoft.Azure.Management.ResourceManager.Fluent</AssemblyName>
     <PackageId>Microsoft.Azure.Management.ResourceManager.Fluent</PackageId>
     <PackageTags>Microsoft Azure Resource Manager;resource;Fluent</PackageTags>
   </PropertyGroup>
-  
   <PropertyGroup>
-    <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
+    <TargetFrameworks>net452;net461;netstandard1.4</TargetFrameworks>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Rest.ClientRuntime.Azure.Authentication" Version="2.3.2" />
+    <PackageReference Include="Microsoft.Rest.ClientRuntime.Azure.Authentication" Version="2.3.3" />
```

Comment:
Please also upgrade the Auth package version in `Tests\AzSdk.test.reference.props` file.